docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_layouts.scss

```diff
@@ -0,0 +1,159 @@
+$hb-sidebar-width: 16.5rem;
+
+.hb-three-column-w-image,
+.hb-three-column {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 1rem 3rem;
```

Comment:
Add any margins as variables in `/settings/variables.general.scss`. If we do need them to behave differently at different breakpoints then we can make a scss map.